docs/03-usage.md

```diff
@@ -29,19 +29,31 @@ Remember to re-run Snowpack every time you import an new dependency.
 ### Whitelisting Dependencies
 
 ```js
-  /* package.json */
-  "snowpack": {
-    "webDependencies": [
-      "htm",
-      "preact",
-      "preact/hooks", // A package within a package
-      "unistore/full/preact.es.js", // An ESM file within a package (supports globs)
-      "bulma/css/bulma.css" // A non-JS static asset (supports globs)
-    ],
-  }
+/* package.json */
+"snowpack": {
+  "webDependencies": [
+    "htm",
+    "preact",
+    "preact/hooks", // A package within a package
+    "unistore/full/preact.es.js", // An ESM file within a package (supports globs)
+    "bulma/css/bulma.css" // A non-JS static asset (supports globs)
+  ],
+}
 ```
 
 Optionally, you can also whitelist any dependencies by defining them in your "webDependencies" config (see below). You can use this to control exactly what is installed, including non-JS assets or deeper package resources.
 
 Note that this config will disable the zero-config mode that attempts to install every package found in your package.json "dependencies". Either use this together with the `--include` flag, or just make sure that you whitelist everything that you want installed.
 
+
+### Skipping NPM Install
+
+```
+$ snowpack --source pika
+```
+
+Snowpack defaults to reading packages from your local `node_modules/` directory, but you can control this behavior with the `--source` flag to simplify your Snowpack install step even further.
+
+Use `--source pika` to have Snowpack fetch packages directly from the [Pika CDN](https://cdn.pika.dev/) instead of your local `node_modules/` directory. Your app will still get the same npm package code, but now Snowpack is fully installing and managing your npm dependencies for you. **With `--source pika` you no longer have to run `npm install` before running Snowpack.**
```

Comment:
Does the Pika CDN have any other benefits to using this? Or is the first step mainly speed? At least in my tests, I noticed this to be pretty snappy, and the caching seemed to work just fine. I’m wondering if it’s worth calling more attention to the faster Snowpack build times here. Or if there’s some other benefit I’m not thinking of (my brain went to some of the Pika CDN browser-specific stuff, but I think that doesn’t apply for build-time Snowpack, right?).

Reply:
Yea, speed and simplicity are the two main benefits here. Pika CDN can do a ton of cool stuff when you point your site to it, but in this context, we're just using it to get the modern JS code and then pass it through Snowpack's install (build) step.